### ISSUE TITLE
Enchantment name bug 

### ISSUE THREAD

**Issue by EternalOne11** [Block-Vader/Enchantment-tweaks#12]
When enchantment tweaks is installed it adds glitched characters in front of all enchant names.
![2019-06-08_11 39 00](https://user-images.githubusercontent.com/42950385/63972018-d67cd380-ca75-11e9-97ca-f74633cadec4.png)

And yes I have tested should have made this crash report 1 month ago

**Comment by Tohricat** [comment]
Might be a problem with SmoothFonts. I'm also having the same issue.

**Comment by EternalOne11** [comment]
no, it also happens when I have no texture the problem is when I took those pictures I had a texture pack even though it was months before I made this issue. and what do you mean by smoothFonts?

**Comment by G3ra1d13** [comment]
found that in configs if you turn off the rarity color designation it removes the glyphs from in front of the name. it is most likely a mistake in the actual unicode characters for color designation...

**Comment by SmileycorpMC** [comment]
ClientEventHandler.java is saved with ANSI encoding, minecraft text uses UTF-8, which causes all uses of '§' in the file to be replaced with a different character.
